test/src/RunCanvas.cpp

```diff
@@ -163,6 +163,9 @@ class TestWindow final : public UI::SingleWindow {
       canvas.DrawPolygon(p2, 3);
       label = _T("huge polygon");
       break;
+
+    default:
```

Comment:
When page is larger than 6 by accident or error label is undefined.
Improved with
`default:
    gcc_unreachable();`
to indicate to gcc that an uninitialized valiable is no possibility here, an page will alway be in the expected range.
Otherwise the program will stop very hard.

Reply:
This accident would be a bug, and I don't want to hide bugs.

Reply:
> This accident would be a bug, and I don't want to hide bugs.

I will throw a runtime_error exception instead.